graphql_api/tests/test_account.py

```diff
@@ -318,7 +226,7 @@ def test_fetch_organizations_pagination(self) -> None:
             for node in result["owner"]["account"]["organizations"]["edges"]
         ]
 
-        assert orgs == ["owner-2", "owner-1"]
```

Comment:
how come this changed?

Reply:
I think Spencer reversed the ordering from desc to asc and it's limited to `first: 2`

Reply:
yep that